schema/4.0.0/schema.md

```diff
@@ -1112,6 +1112,72 @@ See also `default_embedding` in `uns`.
 </tbody></table>
 <br>
 
+
+### {column}_colors
+
+<table><tbody>
+  <tr>
+    <th>Key</th>
+      <td>
+        {column}_colors where {column} MUST be the name of a <code>category</code> data type column in <code>obs</code> that<br> is annotated by the data submitter or curator. The following columns that are annotated by CELLxGENE<br> Discover MUST NOT be specified as {category}:<br><br>
+      <ul>
+        <li>assay</li>
+        <li>cell_type</li>
+        <li>development_stage</li>
+        <li>disease</li>
+        <li>organism</li>
+        <li>self_reported_ethnicity</li>
+        <li>sex</li>
+        <li>tissue</li>       
+      </ul><br>
+      Instead annotate {column}_ontology_term_id_colors for these columns such as <code>assay_ontology_term_id</code>.<br><br>
+    </td>
+  </tr>
+  <tr>
+    <th>Annotator</th>
+    <td>Curator</td>
+  </tr>
+  <tr>
+    <th>Value</th>
+      <td>
+        <code>numpy.ndarray</code>. This MUST be a 1-D array of shape <code>(, c)</code>, where <code>c</code> is greater than or equal to the<br> number of unique categories in the {column} as calculated by:<br><br>
+           <samp>len(anndata.obs.{category}.unique())</samp><br><br>
+        The color code at the Nth position in the <code>ndarray</code> corresponds to the Nth category of <samp>anndata.obs.{category}.unique()</samp>.<br><br>For example, if <code>cell_type_ontology_term_id</code> includes two unique categories:<br><br>
```

Comment:
anndata.obs.{~category~column}.unique()

Reply:
Addressed in 0f62f4ed2625740ab6cbfc35d6f32123fd2716d4.